pheps/phep-9999.md

```diff
@@ -0,0 +1,162 @@
+| DOI | Required | Required | Required | Not Required |
+| pip installable? | Yes | Yes | Yes | No |
+| conda installable? | Yes | No | No | No |
+
+Descriptions for each heading are as follows:
+   - Self Evaluation Status: indicates whether a package has completed a self evaluation against PyHC's standards
+   - PyHC Standard Grades: indicates status of each standards grouping within a package's self evaluation
+   - pyOpenSci Review Status: indicates status of a pyOpenSci review
+   - PyHC Env Installation Conflicts: indicates state of installation conflicts within the PyHC software environment (i.e. does the Package comply with PHEP 3)
+   - HSSI Metadata Compliant: indicates state of compliance with the new Heliophysics Software Search Interface's metadata requirements
+   - Software License: indicates level of software license compliance with the [PyHC Standards Document](https://doi.org/10.5281/zenodo.2529131) 
+   - DOI: indicates whether or not a package's software repository has a DOI (e.g., from Zenodo) 
+   - pip installable?: indicates whether a package is pip installable
+   - conda installable?: indicates whether a package is conda installable
+
+The following table shows the benefits that are associated with each tier:  
+
+|       | **Gold** | **Silver** | **Bronze** | **Copper** | 
+| :-: | :----------: | :-----------: | :------------: | :-------------: |
+| Summer School Inclusion * | Yes | Yes** | No | No |
+| PyHC-top-tier Env Inclusion * | Yes | Yes ** | No | No | 
+| PyHC-all Env Inclusion * | Yes | Yes | Yes | No |
+| PyHC-Chat Bot Inclusion * | Yes | Yes ** | No | No |
+| pyOpenSci Verified Badge | Yes | No | No | No |
+| Standards Compliance Assistance * | Yes *** | Yes | Yes | Yes |
+| Main Page Listing | Yes | Yes | Yes | No |
+| Secondary Page Listing | No | No | No | Yes |
+| HSSI Inclusion | Yes | Yes | Yes | Yes | 
+| Conference Travel Funding | Yes | Yes | Yes | No |  
+
+*  = can opt-in or opt-out of this  
+
+** = conditional upon justification on importance/use of package (e.g. number of users, level of commitment within PyHC activities, or effort made for those items but gold level not yet achieved)  
+
+*** = conditional upon justification that package is in danger of dropping down to a Silver tier 
+
+Descriptions for each heading are as follows:
+   - Summer School Inclusion: indicates whether a package will be included in summer school teaching materials
+   - PyHC-top-tier Env Inclusion: indicates whether a package will be included within the current PyHC software environment used at the summer school (also included within env in Science Platforms Coordination group???)
+   - PyHC-all Env Inclusion: indicates whether a package will be included within the current PyHC software environment containing all packages Bronze and higher. (Does this make sense based on the ability or not of a package to be installed in a common software env???)
```

Comment:
That question in the parentheses is for you @sapols :D

Reply:
Yes it makes sense. I'd say we can remove this question 🙂

Reply:
Well, one thing with the Silver and Bronze levels is that they are allowed to have installation conflicts. Maybe that should change then?  Though, are there other kinds of issues that can occur with integrating a package into the environment other than an installation conflict? @sapols

Reply:
Nevermind on that one, I made a change to the previous chart that makes it make more sense. ha